src/scenes/Dashboard/index.js

```diff
@@ -33,7 +33,7 @@ Dashboard.propTypes = {
 };
 
 Dashboard.defaultProps = {
-  sortBy: undefined,
+  sortBy: SortBy.Ascending,
```

Comment:
do you think we should have this default in the `reducer` so we dont have to declare a default for each class that uses it?

Reply:
need to think about this. cuz we not only need the sort order for request but also for display. let's do a reorg later?